Update qBittorrent docs for new sensors (#37447)
home-assistant/home-assistant.github.io: source/_integrations/qbittorrent.markdown

The qBittorrent integration will add the following sensors:

- `sensor.qbittorrent_status`: The status of qBittorrent - `up_down`, `seeding`, `downloading`, or `idle`.
- `sensor.qbittorrent_connection_status`: The connection status of qBittorrent - `connected`, `firewalled`, or `disconnected`.
- `sensor.qbittorrent_upload_speed`: The current total upload speed in kB/s.
- `sensor.qbittorrent_download_speed`: The current total download speed in kB/s.
- `sensor.qbittorrent_upload_speed_limit`: The active qBittorrent upload speed limit (disabled by default).
- `sensor.qbittorrent_download_speed_limit`: The active qBittorrent download speed limit (disabled by default).
- `sensor.qbittorrent_alltime_upload`: The All-time uploaded data amount.
- `sensor.qbittorrent_alltime_download`: The All-time downloaded data amount.

> - `sensor.qbittorrent_global_ratio`: The global ratio of all torrents (disabled by default).

Reviewer: ```suggestion
- `sensor.qbittorrent_global_ratio`: The global upload-to-download ratio of all torrents (disabled by default). A positive number means there have been more uploads than downloads.
```

Author: That's still not fully correct. The ratio can't go below 0.00 (at least for per-torrent ratio) and I don't think it can for the global ratio either.

I think we should just describe it like it's shown in qBittorrent itself, that should be enough: `The global All-time share ratio (disabled by default).`
The people that use torrents and care about the ratio will know what it is. It's general basic knowledge about torrenting.

Thanks for your help tho, english is not my native language :)

The qBittorrent integration will add the following sensors:

> - `sensor.qbittorrent_status`: The status of qBittorrent - `up_down`, `seeding`, `downloading` or `idle`.

Reviewer: ```suggestion
- `sensor.qbittorrent_status`: The status of qBittorrent - `up_down`, `seeding`, `downloading`, or `idle`.
```

> - `sensor.qbittorrent_connection_status`: The connection status of qBittorrent - `connected`, `firewalled` or `disconnected`.

Reviewer: ```suggestion
- `sensor.qbittorrent_connection_status`: The connection status of qBittorrent - `connected`, `firewalled`, or `disconnected`.
```

- `sensor.qbittorrent_upload_speed`: The current total upload speed in kB/s.
- `sensor.qbittorrent_download_speed`: The current total download speed in kB/s.
- `sensor.qbittorrent_upload_speed_limit`: The active qBittorrent upload speed limit (disabled by default).
- `sensor.qbittorrent_download_speed_limit`: The active qBittorrent download speed limit (disabled by default).

> - `sensor.qbittorrent_alltime_upload`: The All-time uploaded data amount.

Reviewer: ```suggestion
- `sensor.qbittorrent_alltime_upload`: The total amount of data that has been uploaded since qBittorrent has been installed.
```
I'm rephrasing this a bit to make it clearer what "all-time" means. Please correct me if I got this wrong.

Author: It's the actual total transferred data since qBittorrent has been installed. It's not related to the integration installation time.

Reviewer: Ah, ok. Thank you for explaining this.

> - `sensor.qbittorrent_alltime_download`: The All-time downloaded data amount.

Reviewer: ```suggestion
- `sensor.qbittorrent_alltime_download`:  The total amount of data that has been downloaded since qBittorrent has been installed.

Author: Same here

> - `sensor.qbittorrent_global_ratio`: The global ratio of all torrents (disabled by default).

Reviewer: What do you mean by global ratio? Ratio of what?

Author: It's the ratio between download/upload of all torrents combined.
That's how it's displayed in qBittorrent itself: 
![Screenshot_20250215_113749](https://github.com/user-attachments/assets/95b580d5-8e02-4acd-88f4-00fc89eb89bf)

Reviewer: thx! updated the description. let me know if it is correct.
